feddlib/core/FE/Helper.cpp

```diff
@@ -1557,6 +1741,111 @@ int Helper::getDPhiAtCM(vec3D_dbl_ptr_Type &DPhi,
 
     }
 
+/*!
+
+\brief Returns neccesary quadrature Values. Is distinguishes between needing Element or Surface information. 
+
+@param[in] dim Dimension for which the quadrature points are needed.
+@param[in] FEType Finite element type for which the quadrature points are needed.
+@param[in] QuadP Vector with coordinates vectors of quadrature points
+@param[in] QuadW Vector to be filled with the quadrature weights accordingly
+@param[in] vec_LO_Type surfaceIDs for which you need the quadrature points.
+@param[in] points The repeated(!) points of current problem to identify the surface node ids. 
+
+@param[out] QuadPts Quadrature points
+@param[out] QuadW Quadrature weights
+
+\brief Keep in mind that elementwise quadPoints are defined on reference element whereas surface quadPoints at hand are defined on the input surface, which is typically not the reference Element. 
+
+*/
+    void Helper::getQuadraturePntsOnSurfaceInPhysicalSpace(int dim, 	
```

Comment:
May as well leave Points in the name i.e. Pnts --> Points. Looks a bit like a spelling mistake since everything else is spelled out fully :P